Submitter: Timothy Legge <timlegge@gmail.com>
Date: 2020-04-04
AnyEvent::I3 appears to remove the ENV settings before
running to execute via qx in test mode with -T triggers an error:

Insecure directory in $ENV{PATH} while running with -T switch at
/builds/timlegge/aports/community/perl-anyevent-i3/src/
AnyEvent-I3-0.17/blib/lib/AnyEvent/I3.pm line 151.

As the module does this as a normal part of its operation and
tests pass local this appears to be due to the tests running via
the aports build tests

--- a/t/01-workspaces.t
+++ b/t/01-workspaces.t
@@ -1,7 +1,7 @@
 #!perl -T
 # vim:ts=4:sw=4:expandtab
 
-use Test::More tests => 3;
+use Test::More skip_all => "AnyEvent::I3 removes ENV which causes test to fail";
 use AnyEvent::I3;
 use AnyEvent;
 
--- a/t/02-sugar.t
+++ b/t/02-sugar.t
@@ -1,7 +1,8 @@
 #!perl -T
 # vim:ts=4:sw=4:expandtab
 
-use Test::More tests => 3;
+use Test::More skip_all => "AnyEvent::I3 removes ENV which causes test to fail";
+
 use AnyEvent::I3;
 use AnyEvent;
 
